packages/features/shell/Shell.tsx

```diff
@@ -308,10 +308,10 @@ function UserDropdown({ small }: { small?: boolean }) {
             {!small && (
               <span className="flex flex-grow items-center truncate">
                 <span className="flex-grow truncate text-sm">
-                  <span className="text-emphasis mb-1 block truncate font-medium leading-none">
+                  <span className="text-emphasis mb-1 pb-1 block truncate font-medium leading-none">
                     {user.name || "Nameless User"}
                   </span>
-                  <span className="text-default block truncate font-normal leading-none">
+                  <span className="text-default pb-1 truncate font-normal leading-none">
```

Comment:
This change itself fixes the issue for me. Line 311 change doesn't seem required.

In both the cases, I am not sure if this is the right solution, adding padding to solve a text not visible issue seems hackish.